### ISSUE TITLE
[Styling] Line width of streets in Basic

### ISSUE THREAD

**Issue by lazaa32** [maptiler/qgis-maptiler-plugin#79]
Streets look quite thick on some zoom levels.

**Comment by lazaa32** [comment]
Not entirely sure it is a bug. We use this style for road_minor:
```

      "filter": [
        "all",
        [
          "==",
          "$type",
          "LineString"
        ],
        [
          "in",
          "class",
          "minor",
          "service"
        ]
      ],
      "id": "road_minor",
      "layout": {
        "line-cap": "round",
        "line-join": "round"
      },
      "minzoom": 13,
      "paint": {
        "line-color": "hsl(0, 0%, 97%)",
        "line-width": {
          "base": 1.55,
          "stops": [
            [
              4,
              0.25
            ],
            [
              20,
              30
            ]
          ]
        }
      },
      "source": "openmaptiles",
      "source-layer": "transportation",
      "type": "line"
    }
```

And parsed to QGIS it is:
`scale_exp(@zoom_level, 4, 20, 0.25, 30, 1.55) * 0.127'}`

**Comment by lazaa32** [comment]
Fix in #101.
